plugins/database/src/db_plugin.cpp

```diff
@@ -14012,19 +14020,22 @@ namespace
                     cllBindVars[1] = hostIp;
                     cllBindVarCount = 2;
 
-                    irods::error ret = execute_sql("insert into R_TICKET_ALLOWED_HOSTS (ticket_id, host) values (? , ?)");
+                    irods::error ret =
+                        execute_sql("insert into R_TICKET_ALLOWED_HOSTS (ticket_id, host) values (? , ?)");
                     if (!ret.ok()) {
                         log_sql::error("Failed to add host [{}] to ticket [{}], status [{}]",
-                                        hostIp,
-                                        ticket_id_string.c_str(),
-                                        ret.code());
+                                       hostIp,
+                                       ticket_id_string.c_str(),
```

Comment:
If `ticket_id_string` is a `std::string` or `std::string_view`, then you drop the use of `.c_str()`.

fmtlib is used inside of the logger, and so it understands how to handle those types.

_This applies to other locations too._